sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -940,7 +940,6 @@ object SparkSession {
         }
 
         session = new SparkSession(sparkContext, None, None, extensions)
-        options.foreach { case (k, v) => session.sessionState.conf.setConfString(k, v) }
```

Comment:
Thank you for review, @gatorsmile .
I see. Then, @cloud-fan meant the whole #18172 intead of that line.